frontend/README.md

```diff
@@ -0,0 +1,14 @@
+# URL-S Frontend
+
+Instructions for configuring and starting the URL-S frontend app, Vue.js, and its dependencies.
+
+It is recommended that you install and start the Backend applications before using the Frontend. See the README in the Backend folder.
+
+## Local Setup
+**`Step 1`**. Open a terminal (option to do this in MS Code). Change directory to the frontend/url-shortener folder
```

Comment:
lets be clear on the folder here as at a first glance i tried to do the npm install in front-end folder, but it needs to be done one folder inside it.